src/vmm/src/builder.rs

```diff
@@ -308,11 +644,23 @@ pub fn build_microvm_for_boot(
     }
 
     #[cfg(target_arch = "aarch64")]
-    attach_legacy_devices_aarch64(event_manager, &mut vmm, &mut boot_cmdline).map_err(Internal)?;
+    aarch::attach_legacy_devices_aarch64(event_manager, &mut vmm, &mut boot_cmdline)
```

Comment:
`s/aarch/aarch64` in the module name

Reply:
I meant `aarch::attach_legacy_devices` -> `aarch64::attach_legacy_devices` since `aarch64` is the module name.